schema/deploy/mutations/custom_create_naics_mutation.sql

```diff
@@ -0,0 +1,20 @@
+-- Deploy ggircs-portal:mutations/custom_create_naics_mutation to pg
+-- requires: tables/naics
+
+begin;
+
+create or replace function ggircs_portal.custom_create_naics_mutation(
```

Comment:
can we name it `create_naics_mutation`? I guess we don't want to use the standard postgraphile mutation, so it would be good if our function was to override it

Reply:
I think there's maybe value in making the name different because then it's easy to see in the front-end mutation definition that we are using a custom function from our database rather than the default. I suppose comments in the front-end mutation would be enough, as long as we remember to write them. Thoughts?

Reply:
Yeah but we never want to use the default mutation, right? So if we can override it (or omit the generation of the default create mutation for this table), then there is no ambiguity when looking at the graphql API

Reply:
Cool, makes sense. I'll make a note in the card to add some comments so it's still clear that we're using a custom mutation